tests/wsproxy/conftest.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from typing import Any, Dict
+from uuid import UUID
+
+import pytest
+
+from ai.backend.wsproxy.proxy.frontend.http.port import PortFrontend
+from ai.backend.wsproxy.types import AppMode, Circuit, FrontendMode, ProxyProtocol, RouteInfo
+
+
+class DummyRequest:
+    def __init__(self, app_data: Dict[str, Any]):
+        self.app = app_data
+        self._state: Dict[str, Any] = {}
+
+    def __getitem__(self, key: str):
+        return self._state[key]
+
+    def __setitem__(self, key: str, value: Any):
+        self._state[key] = value
+
+
+@pytest.fixture
+def dummy_request():
+    def _dummy_request(app_data: Dict[str, Any]):
+        return DummyRequest(app_data)
+
+    return _dummy_request
```

Comment:
- Removed specific fixtures from conftest.py, leaving only the general **create_circuit** fixture. 
(They can be moved back if needed in the future.)

- Modified the `create_circuit` function to accept more general parameters, allowing it to create a `Circuit` with customizable values. 
(Future test cases can pass the appropriate arguments based on their specific requirements.)

Reply:
I still don't recognize any benefits of leaving `create_circuit` as a fixture populating an inner function. What would be the upsides of doing so instead of just exposing inner function itself as a role of helper function?

Reply:
The create_circuit function is currently only used in test_port.py, so I will include it there for now.

If this function is needed in other test code in the future, we can move it to utils.py. (However, if it’s added under the tests directory, we'll need to address the PYTHONPATH issue!)